src/main/java/TdpRecruitmentApplication.java

```diff
@@ -1,4 +1,3 @@
-import com.github.dirkraft.dropwizard.fileassets.FileAssetsBundle;
```

Comment:
Maybe it is better to leave this import as it is in master, it might be unnecessary but it might also cause conflicts